src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -4,6 +4,7 @@
 			<div class="column small-12">
 				<h2 class="category-name">
 					<router-link
+						v-if="showViewAllLink"
```

Comment:
For the category row title this else statement makes it so we just show a title without a link!